types/2020-08-27/BillingPortal/Configurations.d.ts

```diff
@@ -25,7 +25,7 @@ declare module 'stripe' {
         /**
          * ID of the Connect Application that created the configuration.
          */
-        application: string | null;
+        application: string | Stripe.Application | null;
```

Comment:
This is technically breaking? Are we allowing this in TS?

Reply:
Yes, that is the present policy.